system-tests/management-api/management-api-test-runner/build.gradle.kts

```diff
@@ -33,6 +33,8 @@ dependencies {
     testImplementation(libs.awaitility)
     testImplementation(libs.junit.jupiter.api)
     testImplementation(libs.jakartaJson)
+    // used for the constants
+    testImplementation(project(":extensions:control-plane:api:management-api:contract-definition-api"))
```

Comment:
as they are e2e tests, maybe they could not rely on constants and use the actual strings